File: geeksforgeeks/array/rotation_method_one.py
arr = [1,2,3,4,5,6,7,8]
n = 3
for i in range(0, n):
    first = arr[0]
    for y in range(0, len(arr)-1):
        arr[y] = arr[y+1]
    arr[len(arr)-1] = first
    
print(arr)

File: geeksforgeeks/array/rotation_method_one.cpp
#include <bits/stdc++.h>

using namespace std;

int main()
{
    int arr[] = {1,2,3,4,5,6,7};
    int len = sizeof(arr)/sizeof(arr[0]);
    int n = 3;
    for (int i =0; i < n; i++){
        int first = arr[0];
        for (int j=0;j<len; j++){
            arr[j] = arr[j+1];
        }
        arr[len-1] = first;
    }
    for(int k = 0; k < len; k++){
        cout << arr[k] << " ";
    }
    return 0;
}